### DISCUSSION TITLE
Gerätelabel: Datum einfügen

### DISCUSSION THREAD

**Discussion by epiphysus** [Zefau/ioBroker.jarvis#1788] [Q&A]
Ich habe manche Popups so eingerichtet, das sie künftige Wetterwerte anzeigen. (Regen morgen. = x,  Regen übermorgen = y...
gerne würde ich statt "übermorgen, überübermorgen" den Wochentag sehen. Meine Frage; ist es möglich im Feld Label der Geräte einen Link auf einen Wert zu setzen, der somit dann als flexibler Label im Popup erscheint?

**Comment by mcuiobroker** [comment]
Ja ist mit v3.1.x möglich. Man kann dort ein Binding definieren. Dies bezieht sich dann auf einen Wert (DP) in ioBroker.
z.B.  `{"default":"{0_userdata.0.jarvis.test.text}"}`
![image](https://user-images.githubusercontent.com/84818934/180126517-83861f51-17a4-4da0-a7a6-130477839ffb.png)
-> 
![image](https://user-images.githubusercontent.com/84818934/180126583-fa382190-2416-4e8f-b204-572d9635af6d.png)

**Comment by epiphysus** [comment]
Danke für die Antwort. Darauf freue ich mich schon sehr.
